include/deal.II/lac/linear_operator.h

```diff
@@ -821,6 +862,50 @@ namespace internal
         v.reinit(matrix.n(), omit_zeroing_entries);
       }
     };
+
+    class EmptyPayload
+    {
+    public:
+      // Default constructor
+      EmptyPayload ()
+      {}
+
+      // Constructor for a matrix type based on an exemplary matrix
+      // It is from this constructor that we can safely
+      // build a operational instance of the payload
+      template<typename MatrixExemplar, typename Matrix>
```

Comment:
For all of the member functions of this class, please use doxygen-style `/** ... */` comments so that doxygen actually picks them up.